conf/ocp_version/ocp-4.4-ga-config.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Config file for OCP GA 4.4 stable channel
+
+RUN:
+  client_version: '4.4-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.4-ga"
+  ocp_channel: "stable"
+ENV_DATA:
+  vm_template: 'rhcos-4.4.0-x86_64-vmware'
```

Comment:
Where do u get above template ?

Reply:
I used the template from ocp-4.3-ga-config.yaml and ocp-4.2-ga-config.yaml

Reply:
https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.4/latest/ there is no such image yet so go with what we have in non GA and make a comment/note about once there will be RHCOS gaed image we should change here and open the issue to ecosystem team please about this which you can mention in the comment.

Thanks

Reply:
Thanks @petr-balogh I changed the version and also added comment once GAed we can change that.